apps/examples/cy4390x_demo/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see misc/tools/kconfig-language.txt.
```

Comment:
same as previous. Please use new sentence

Reply:
it will be changed as your guidance.
BR, HJ Lee.